index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+</body>
```

Comment:
Kindly you should have a `<h1>` characterizing the title in the page body of which your tag should contain a class so that it allows you to select it in your CSS. it is very important to familiarize yourself with this practice according to the project description